lib/app/util/enum/swagger_path_request_type.dart

```diff
@@ -3,6 +3,7 @@ enum SwaggerPathRequestType {
   post,
   put,
   delete,
+  patch,
```

Comment:
@cozvtieg9, one swagger did not work properly and fell on an unimplemented orElse